src/scripts/assets/build.js

```diff
@@ -0,0 +1,6 @@
+const shell = require('shelljs');
+
+shell.exec(`
```

Comment:
`shelljs` isn't needed here, node's native `child_process.execSync` works great.
But since `shelljs` is already a dependency of the projet for some other features it isn't a major issue

Reply:
I'll delete it :)